targets/csharp/make.js

```diff
@@ -50,7 +60,7 @@ function getBaseTypeSyntax(datatype) {
     //begin classes - only 1 possible
     if (datatype.className.toLowerCase().endsWith("request"))
         parents.push("PlayFabRequestCommon");
-    else if (datatype.className.toLowerCase() === "loginresult" || datatype.className === "RegisterPlayFabUserResult")
+    else if (datatype.className.toLowerCase() === "loginresult" || datatype.className === "AuthenticateIdentityResult" || datatype.className === "RegisterPlayFabUserResult")
```

Comment:
Nit:
The "toLowerCase" on LoginResult could probably be removed. I think I thought at the time that it'd be more safe, but so many years later, I think it becomes less safe. The chance of names that legitimately differ only by capitalization is probably expanding. And no other part of the pipeline would be limited in this way, making THIS the [only?] point of failure. (And anything else that uses toLowerCase() in this way.)